### PR TITLE
don't claim control over the oauth and user API

### PR DESCRIPTION
**Pull request by stlaz**
Removes the oauth and user APIs from the list of
apiservices.

/assign @p0lyn0mial 

----

Seems straightforward, not sure if there are prereqs, but I think we should be fine?

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -172,14 +171,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	).WithAPIServiceController(
 		"openshift-apiserver",
-		apiservice.NewAPIServicesToManage(
```
Comment (p0lyn0mial) [inline pkg/operator/starter.go:175]: we should also remove https://github.com/openshift/cluster-openshift-apiserver-operator/blob/d54f7de5ad41056faa9d30e6e2e306ec5b850ecc/pkg/operator/apiservice/apiservice_openshiftapiserver_sync.go

**Comment by p0lyn0mial** [comment]
/hold

I think we should remove one more file https://github.com/openshift/cluster-openshift-apiserver-operator/pull/417#discussion_r529293329

File: pkg/operator/starter.go (hunk 1 above)
Comment (stlaz) [reply pkg/operator/starter.go:175]: I thought I did that, should be gone now :+1:

**Comment by stlaz** [comment]
/retest
failures seem unrelated

**Comment by p0lyn0mial** [comment]
/hold

failures look weird, looking - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-openshift-apiserver-operator/417/pull-ci-openshift-cluster-openshift-apiserver-operator-master-e2e-aws-operator/1331918773271138304/artifacts/e2e-aws-operator/test/junit_report.xml

**Comment by stlaz** [comment]
/retest
still probably unrelated

**Comment by stlaz** [comment]
/hold
issue identified - let's have it keep bashing its head against CI, maybe it'll pass one day

**Pull request merged**